metagraph-plugin/README.md

```diff
@@ -0,0 +1,77 @@
+# Metagraph Interoperability for Katana Graph
+
+
+Overview
+========
+
+Metagraph is a plugin-based system for performing graph computations;
+Users can use graph algorithms/representations from different graph libraries with the same API interface.
+
+Highlights include:
+- Katana Graph built-in algorithms/types which can be used by existing Metagraph API.
+- Graph analytics algorithms supported by both existing metagraph plugin graph libraries and Katana Graph.
+- The bi-direction translation between Katana Graph format and the NetworkX Graph format.
+
+
+
+Installation
+===============
+
+First follow this repo [Katana Graph](https://github.com/KatanaGraph/katana) to install the python environment. Then run:
+
+```Shell
+$BUILD_DIR/python_env.sh python setup.py develop
+```
+
+to install the plugin.
```

Comment:
This is the only way to install at this point from what I can tell. That's fine for the moment.

Please contact me and I can setup conda packaging or help you set it up. We can do than in an upcoming PR. We don't want people to have to build anything to install this.

Reply:
Oh and we can update these instructions (in another PR) to show how to build this package without building the rest of Katana and instead using the conda packages for katana.